tools/ci_build/build.py

```diff
@@ -46,9 +46,12 @@ def __init__(self, message):
 
 
 def _check_python_version():
-    if (sys.version_info.major, sys.version_info.minor) < (3, 7):
+    # TODO Upgrade this, Python 3.6 is no longer supported. However, some packaging pipelines are still using it.
```

Comment:
Can we update the pipelines? The failing post-merge job was using python 3.8 so I'm not sure which pipeline uses 3.6 and why.

Reply:
we should. I am not sure where that change should be made to update the packaging pipeline to use python 3.7+. any pointers @mszhanyi?

Reply:
@edgechen1
Currently, the range of Python versions that onnxruntime support is form 3.8 to 3.11
In Azure pipelie, [UsePythonVersion](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/use-python-version-v0?view=azure-pipelines) to download or select a version of Python to run on an agent.

Reply:
this one's from one of the centos docker images. I think it gets installed here:

https://github.com/microsoft/onnxruntime/blob/b4c4e2b594b440124436650dc41949413d85a59d/tools/ci_build/github/linux/docker/inference/x64/default/cpu/scripts/install_centos.sh#L8

not too familiar with this docker image. how can we change the Python version here?

Reply:
I'll update Python in another PR